AEPMessaging/Tests/UnitTests/Event+MessagingTests.swift

```diff
@@ -52,15 +52,19 @@ class EventPlusMessagingTests: XCTestCase {
     // MARK: - Helpers
 
     /// Gets an event to use for simulating a rules consequence
-    private func getRulesResponseEvent(type: String? = MessagingConstants.ConsequenceTypes.IN_APP_MESSAGE,
+    private func getRulesResponseEvent(type: String? = MessagingConstants.ConsequenceTypes.SCHEMA,
                                triggeredConsequence: [String: Any]? = nil,
                                removeDetails: [String]? = nil) -> Event {
 
         // details are the same for postback and pii, different for open url
-        var details: [String: Any] = type == MessagingConstants.ConsequenceTypes.IN_APP_MESSAGE ? [
-            MessagingConstants.Event.Data.Key.IAM.TEMPLATE: MessagingConstants.Event.Data.Values.IAM.FULLSCREEN,
-            MessagingConstants.Event.Data.Key.IAM.HTML: testHtml,
-            MessagingConstants.Event.Data.Key.IAM.REMOTE_ASSETS: testAssets
+        var details: [String: Any] = type == MessagingConstants.ConsequenceTypes.SCHEMA ? [
+            MessagingConstants.Event.Data.Key.ID: mockMessagingId,
+            MessagingConstants.Event.Data.Key.SCHEMA: MessagingConstants.PersonalizationSchemas.IN_APP,
+            MessagingConstants.Event.Data.Key.DATA: [
+                "contentType": MessagingConstants.ContentTypes.TEXT_HTML,
+                "content": testHtml,
+                "remoteAssets": testAssets
```

Comment:
nit: here too we have mixed usage of hardcoded keys and constants. I believe you're hoping for us to fix this when we add more tests.